paint-timing/fcp-only/fcp-visibility.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<head>
+<title>Performance Paint Timing Test: FCP due to visibility change</title>
```

Comment:
This seems like a repeated test of the two-steps one?

Reply:
ok